java/arcs/core/data/proto/manifest.proto

```diff
@@ -120,6 +122,12 @@ message CollectionTypeProto {
   TypeProto collection_type = 1;
 }
 
+message SingletonTypeProto {
+  TypeProto singleton_type = 1;
+}
+
+message CountTypeProto {}
```

Comment:
Is this a kosher proto?

Reply:
hahaha, I don't know... Do we actually need it, or is it here just for completeness?

Reply:
It's here for completeness. There are only a few use cases that I've found for `CountType` -- it seems to be used in Stores. However, this is a type specified in `Tag.kt`.